src/DotNet.Status.Web/DotNet.Status.Web.csproj

```diff
@@ -20,11 +20,16 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" />
     <PackageReference Include="Octokit" />
     <PackageReference Include="WindowsAzure.Storage" /> 
+    <Compile Remove="tests\**" />
+    <Content Remove="tests\**" />
```

Comment:
I don't think all these `Remove="tests\**"` are necessary now that we moved the tests to a sibling folder.

Reply:
Had to manually clean this up since `git merge` from master didn't make a difference.